tests/RobotFramework/tasks/E2E_Tutorials/connect_to_c8y_send_measurements.robot

```diff
@@ -0,0 +1,173 @@
+    Log    ${output}
+    Should Contain    ${output}    Checking if systemd is available.
+    Should Contain    ${output}    Checking if configuration for requested bridge already exists.
+    Should Contain    ${output}    Validating the bridge certificates.
+    Should Contain    ${output}    Creating the device in Cumulocity cloud.
+    Should Contain    ${output}    Saving configuration for requested bridge.
+    Should Contain    ${output}    Restarting mosquitto service.
+    Should Contain    ${output}    Awaiting mosquitto to start. This may take up to 5 seconds.
+    Should Contain    ${output}    Enabling mosquitto service on reboots.
+    Should Contain    ${output}    Successfully created bridge connection!
+    Should Contain    ${output}    Sending packets to check connection. This may take up to 2 seconds.
+    Should Contain    ${output}    Connection check is successful.
+    Should Contain    ${output}    Checking if tedge-mapper is installed.
+    Should Contain    ${output}    Starting tedge-mapper-c8y service.
+    Should Contain    ${output}    Persisting tedge-mapper-c8y on reboot.
+    Should Contain    ${output}    tedge-mapper-c8y service successfully started and enabled!
+    Should Contain    ${output}    Enabling software management.
+    Should Contain    ${output}    Checking if tedge-agent is installed.
+    Should Contain    ${output}    Starting tedge-agent service.
+    Should Contain    ${output}    Persisting tedge-agent on reboot.
+    Should Contain    ${output}    tedge-agent service successfully started and enabled!
+    Log    Connected to Cumulocity IoT and verified connection for ${DEVICE_SN}
+
+Send Temperature Measurement
+    [Documentation]    Send a temperature measurement.
+    Execute Command    tedge mqtt pub 'te/device/main///m/environment' '{"temperature": 25}'
+    Log    Sent temperature measurement: 25°C
+
+Send Three Phase Current Measurement
+    [Documentation]    Send a three-phase current measurement.
+    Execute Command    tedge mqtt pub 'te/device/main///m/electrical' '{"three_phase_current": {"L1": 9.5, "L2": 10.3, "L3": 8.8}}'
+    Log    Sent three-phase current measurement
+
+Send Combined Measurement
+    [Documentation]    Send a combined measurement.
+    Execute Command    tedge mqtt pub 'te/device/main///m/combined' '{"time": "2020-10-15T05:30:47+00:00", "temperature": 25, "current": {"L1": 9.5, "L2": 10.3, "L3": 8.8}, "pressure": 98}'
+    Log    Sent combined measurement
+
+Send Child Device Temperature Measurement
+    [Documentation]    Send a temperature measurement to a child device.
+    Execute Command    tedge mqtt pub 'te/device/child1///m/environment' '{"temperature": 25}'
+    Log    Sent temperature measurement to child device: 25°C
+
+Verify Measurement In Cumulocity
+    [Arguments]    ${type}    ${value}
+    [Documentation]    Verify the measurement in Cumulocity.
+    # Add implementation for checking the measurement in Cumulocity
```

Comment:
This is a TODO, so it might make sense to add it in the PR (since we already have an assertion for this included in the `Cumulocity` RobotFramework library).

Reply:
Is this task the right place for it?

Reply:
Yes it makes sense to add it since we are taking about a new file, so we shouldn't be adding active TODOs to new files. Either remove the step completely or add the check...But it should be a simple `Device Should Have Measurements    type=environment    minimum=1    maximum=1` (updating to fit the measurements being created).